src/main/java/org/jboss/forge/arquillian/command/algeron/PactAlgeronConsumerTestSetup.java

```diff
@@ -0,0 +1,102 @@
+package org.jboss.forge.arquillian.command.algeron;
+
+import au.com.dius.pact.consumer.dsl.PactDslWithProvider;
+import au.com.dius.pact.model.PactFragment;
+import org.arquillian.algeron.pact.consumer.spi.Pact;
+import org.arquillian.algeron.pact.consumer.spi.PactVerification;
+import org.jboss.forge.roaster.model.source.JavaClassSource;
+import org.jboss.forge.roaster.model.source.MethodSource;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class PactAlgeronConsumerTestSetup implements AlgeronConsumerTestSetup
+{
+
+   private static final String FRAGMENT_METHOD = "" +
+           "final Map<String, String> headers = new HashMap<>();" + System.lineSeparator() +
+           "headers.put(\"Content-Type\", \"application/json\");" + System.lineSeparator() +
+           System.lineSeparator() +
+           "      return builder" + System.lineSeparator() +
+           "              .uponReceiving(\"%s\")" + System.lineSeparator() +
+           "              .path(\"/\")" + System.lineSeparator() +
+           "              .method(\"GET\")" + System.lineSeparator() +
+           "              .headers(headers)" + System.lineSeparator() +
+           "              .willRespondWith()" + System.lineSeparator() +
+           "              .status(200)" + System.lineSeparator() +
+           "              .body(\"\")" + System.lineSeparator() +
+           "              .toFragment();";
+
+   @Override
+   public JavaClassSource updateTest(JavaClassSource test, String consumer, String provider, String fragmentName)
+   {
+
+      addImports(test);
+      annotateTest(test, consumer, provider);
+      createFragment(test, fragmentName);
+      createTest(test, consumer, provider, fragmentName);
```

Comment:
The flow of this methods is a bit counter intuitive. We should probably first create test, then fragment and annotation.